survey/questions/translation_first_draft.md

```diff
@@ -57,27 +69,27 @@ Basic school
 
 High school
 
-Technician
+Technician / Associates
```

Comment:
In the USA this is known as an Associates degree ! I'm not sure if there are other names for it ?